### ISSUE TITLE
Which versions of node are to be supported and tested?

### ISSUE THREAD

**Issue by ccpandhare** [publiclab/image-sequencer#45]
In .travis.yml :
```
language: node_js
node_js:
  - '4'
  - '5'
env:
  - CXX=g++-4.8
```

Are these the only Node versions we should be testing builds on? what about v6, v7, v8?
I run v7.4.0 on my computer.
```
➜  image-sequencer git:(documentation) node -v
v7.4.0
```

**Comment by jywarren** [comment]
sure, try opening a new PR with them!

**Comment by ccpandhare** [comment]
fixed in #46
